docs/tutorials/index.rst

```diff
@@ -41,11 +41,11 @@ Clickstream Data Analysis Pipeline
     
     - :ref:`ksql_clickstream-docker`
 
+    If you do not have Docker, you can also run an `automated version <https://github.com/confluentinc/quickstart-demos/tree/master/clickstream>`_ of the Clickstream tutorial designed for local Confluent Platform installs. Running the Clickstream demo locally without Docker requires that you have Confluent Platform 4.1 installed locally, along with Elasticsearch and Grafana.
```

Comment:
i think you can leave out `4.1` since these docs are versioned - and if we put this in, we'll need to keep up-to-date for each subsequent version.